styx-standalone-service/src/main/java/com/spotify/styx/StyxService.java

```diff
@@ -20,18 +20,31 @@
 
 package com.spotify.styx;
 
+import com.google.common.annotations.VisibleForTesting;
 import com.spotify.apollo.AppInit;
+import com.spotify.apollo.Environment;
 import com.spotify.apollo.httpservice.HttpService;
 import com.spotify.apollo.httpservice.LoadingException;
+import com.spotify.metrics.core.SemanticMetricRegistry;
+import com.spotify.styx.monitoring.MetricsStats;
+import com.spotify.styx.monitoring.Stats;
+import java.time.Instant;
 
 public class StyxService {
 
+  private static Stats STATS;
+
   private StyxService() {
+    throw new UnsupportedOperationException();
   }
 
   public static void main(String[] args) throws LoadingException {
-    final StyxScheduler scheduler = StyxScheduler.createDefault();
-    final StyxApi api = StyxApi.createDefault();
+    final StyxScheduler scheduler = StyxScheduler.newBuilder()
```

Comment:
We can avoid the syncronized static stats holder by moving this into the AppInit lambda.

Create the `MetricsStats` once there and pass it into the api and scheduler builders using lambdas.

Reply:
Yeah that would look nicer. The builder stuff shouldn't have any side effect to make `AppInit` sad. I will fix.